Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/bbox.h

```diff
@@ -255,6 +259,102 @@ namespace CGAL {
       }
       return bb;
     }
+
+    /*!
+    * \ingroup PkgPolygonMeshProcessingRef
+    *
+    * adds an axis-aligned bounding box to a polygon mesh.
+    *
+    * @tparam PolygonMesh a model of `MutableFaceGraph`
+    * @tparam NamedParameters a sequence of \ref bgl_namedparameters "Named Parameters"
+    *
+    * @param pmesh a polygon mesh
+    * @param np an optional sequence of \ref bgl_namedparameters "Named Parameters" among the ones listed below
+    *
+    * \cgalNamedParamsBegin
+    *   \cgalParamNBegin{bbox_scaling}
+    *     \cgalParamDescription{a double used to scale the bounding box.
+    *       The default value is 1 and the bounding box is the smallest possible
+    *       axis-aligned bounding box.}
+    *     \cgalParamDefault{1.}
+    *     \cgalParamPrecondition{`bbox_scaling > 0`}
+    *   \cgalParamNEnd
+    *   \cgalParamNBegin{do_not_triangulate_faces}
+    *     \cgalParamDescription{a boolean used to specify if the bounding box faces
+    *       should be triangulated or not.
+    *       The default value is `false`, and faces are triangulated.}
+    *     \cgalParamDefault{false}
+    *   \cgalParamNEnd
+    *   \cgalParamNBegin{vertex_point_map}
+    *     \cgalParamDescription{a property map associating points to the vertices of `pmesh`}
+    *     \cgalParamType{a class model of `ReadWritePropertyMap` with `boost::graph_traits<PolygonMesh>::%vertex_descriptor`
+    *                    as key type and `%Point_3` as value type}
+    *     \cgalParamDefault{`boost::get(CGAL::vertex_point, pmesh)`}
+    *     \cgalParamExtra{If this parameter is omitted, an internal property map for `CGAL::vertex_point_t`
+    *                     must be available in `PolygonMesh`.}
+    *   \cgalParamNEnd
+    *   \cgalParamNBegin{geom_traits}
+    *     \cgalParamDescription{an instance of a geometric traits class providing the functor `Construct_bbox_3`
+    *                           and the function `Construct_bbox_3 construct_bbox_3_object()`.
+    *                           `Construct_bbox_3` must provide `Bbox_3 operator()(Point_3)`
+    *                           where `%Point_3` is the value type of the vertex point map.
+    *                           It must also provide the functor `Construct_midpoint_3` and,
+    *                           the function `Construct_midpoint_3 construct_midpoint_3_object()`.
+    *                           `Construct_midpoint_3` must provide `Point_3 operator()(Point_3, Point_3)`.
+    *                           The types `Vector_3` and `Iso_cuboid_3` must also be provided.}
+    *   \cgalParamNEnd
+    * \cgalNamedParamsEnd
+    *
+    * @see `bbox()`
+    */
+    template<typename PolygonMesh,
+             typename NamedParameters = parameters::Default_named_parameters>
+    void add_bbox(PolygonMesh& pmesh,
```

Comment:
We wonder if wouldn't be better to add an overload to make_hexahedron that takes a bbox as parameter and do_triangulate as named parameter +  a named parameter to PMP::bbox to have the scaling. Then you'll get the same behavior as the following function but with two lines instead of one.